.github/workflows/run-kata-coco-tests.yaml

```diff
@@ -126,6 +152,14 @@ jobs:
         timeout-minutes: 10
         run: bash tests/integration/kubernetes/gha-run.sh deploy-kata-sev
 
+      - name: Install snapshotter
+        timeout-minutes: 10
+        run: bash tests/integration/kubernetes/gha-run.sh install-snaphostter
```

Comment:
is this a typo, should be `install-snapshotter`

Reply:
Fixed.